api_spec.yaml

```diff
@@ -23,7 +23,44 @@ paths:
         content:
           application/json:
             schema:
-              type: object
+              type: array
+              items:
+                type: object
+                properties:
+                  fixed acidity:
+                    type: number
+                  volatile acidity:
+                    type: number
+                  citric acid:
+                    type: number
+                  residual sugar:
+                    type: number
+                  chlorides:
+                    type: number
+                  free sulfur dioxide:
+                    type: int32
+                  total sulfur dioxide:
+                    type: int32
+                  density:
+                    type: number
+                  pH:
+                    type: number
+                  sulphates:
+                    type: number
+                  alcohol:
+                    type: int32
```

Comment:
Why alcohol type is int32? It is number.

Reply:
Agree, fixed.